libs/frontend/util-router/src/components/RouterLink.vue

```diff
@@ -67,11 +67,15 @@ const isExactActive = computed(() => {
   return lastMatch?.name === targetRoute.value?.name
 })
 
-const navigate = () => {
-  if (replace.value) {
-    router.replace(to.value)
-  } else {
-    router.push(to.value)
+const navigate = (e: Event) => {
+  const isExternal = /^(https?:\/\/|www\.)/.test(to.value.path ?? '')
```

Comment:
@abemscac This is a quick hack to fix the issue. There's probably a better way, but it seems straightforward enough to let the `<a>` proceed naturally (e.g. no `preventDefault`) if the link starts with `http` or `www`.